doc/dox_comments/header_files/random.h

```diff
@@ -137,6 +137,36 @@ WOLFSSL_API int  wc_InitRng(WC_RNG*);
 */
 WOLFSSL_API int  wc_RNG_GenerateBlock(WC_RNG*, byte*, word32 sz);
 
+/*!
+    \ingroup Random
+
+    \brief Creates a new random number.
```

Comment:
This function creates a new WC_RNG structure, not a random number.  Users will use a separate WC_RNG API to get random bytes (ex: wc_RNG_GenerateBlock(), etc).